packages/react-storage/src/components/StorageBrowser/controls/TitleControl.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { Title } from '../composables/Title';
+import { ViewElement } from '../context/elements';
+import { ControlProps } from './types';
+import { useResolvedComposable } from './hooks/useResolvedComposable';
+import { useTitle } from './hooks/useTitle';
+
+export const TitleControl = ({
+  className,
+}: ControlProps): React.JSX.Element | null => {
+  const { props } = useTitle();
+  const ResolvedTitle = useResolvedComposable(Title, 'Title');
+
+  if (!props) {
+    return null;
+  }
+
+  const { children, titleClassName } = props;
```

Comment:
See this comment: https://github.com/aws-amplify/amplify-ui/pull/5935/files/8636d6d874d30ab47bb7a62f863d21356394163d#r1813176271
```suggestion
```

Reply:
TIL 👀 , curious what became possible with TS though